libraries/botframework-connector/src/teams/models/mappers.ts

```diff
@@ -548,11 +485,11 @@ export const TeamsMeetingNotificationChannelData = {
     },
 };
 
-export const TeamsMeetingOnBehalfOf: msRest.CompositeMapper = {
-    serializedName: 'TeamsMeetingOnBehalfOf',
+export const MeetingOnBehalfOf: msRest.CompositeMapper = {
```

Comment:
Is there an existing OnBehalfOf object in JS SDK? Asking as the intention is to use the existing OnBehalfOf object for the new channelData object for Meeting (MeetingChannelData) :) From this perspective, naming this one as `OnBehalfOf` makes more sense to me as the intention is more explicit. Wdyt?

Reply:
Update `MeetingOnBehalfOf` to just `OnBehalfOf`.